[2021-12-24 11:12:43,156] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.apply_model 2021-12-24T11:06:07.235593+00:00 [queued]>
[2021-12-24 11:12:43,327] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.apply_model 2021-12-24T11:06:07.235593+00:00 [queued]>
[2021-12-24 11:12:43,339] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-12-24 11:12:43,350] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-12-24 11:12:43,366] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-12-24 11:12:43,463] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): apply_model> on 2021-12-24T11:06:07.235593+00:00
[2021-12-24 11:12:43,577] {standard_task_runner.py:52} INFO - Started process 49 to run task
[2021-12-24 11:12:43,655] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'classification', 'apply_model', '2021-12-24T11:06:07.235593+00:00', '--job-id', '145', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/classification.py', '--cfg-path', '/tmp/tmp80nvehct', '--error-file', '/tmp/tmp4cikk9pm']
[2021-12-24 11:12:43,792] {standard_task_runner.py:77} INFO - Job 145: Subtask apply_model
[2021-12-24 11:12:44,465] {logging_mixin.py:104} INFO - Running <TaskInstance: classification.apply_model 2021-12-24T11:06:07.235593+00:00 [running]> on host 89d4e2d4fc9f
[2021-12-24 11:12:45,030] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=classification
AIRFLOW_CTX_TASK_ID=apply_model
AIRFLOW_CTX_EXECUTION_DATE=2021-12-24T11:06:07.235593+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-24T11:06:07.235593+00:00
[2021-12-24 11:13:31,018] {logging_mixin.py:104} WARNING - /usr/local/lib/python3.6/site-packages/PIL/Image.py:976 UserWarning: Palette images with Transparency expressed in bytes should be converted to RGBA images
[2021-12-24 11:13:38,598] {python.py:118} INFO - Done. Returned value was: None
[2021-12-24 11:13:38,691] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=classification, task_id=apply_model, execution_date=20211224T110607, start_date=20211224T111243, end_date=20211224T111338
[2021-12-24 11:13:38,935] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-12-24 11:13:39,130] {local_task_job.py:146} INFO - Task exited with return code 0
